### DISCUSSION TITLE
Use fitted model on new unseen dataset without changing parameters

### DISCUSSION THREAD

**Discussion by olerch** [alan-turing-institute/sktime#5006] [General]
I would be interested to know if there is a way to make predictions with a fitted model on a different data set.

Specifically, based on 20 time points in time series A, I want to train an AutoREG model to make a prediction one point into the future. I then want to use the trained model with unchanged parameters to make a prediction of time series B without retraining the parameters.

Here is an example of what I mean:
https://stackoverflow.com/questions/66855043/forecast-with-arima-model-with-python-using-unseen-data-instead-of-training-data
Case C below

**Comment by fkiraly** [comment]
@olerch, currently afaik not without a hack.

This is on our radar though, I think this issue is requesting the same feature: https://github.com/sktime/sktime/issues/4651

FYI @benheid

**Comment by fkiraly** [comment]
Another option could be this: https://github.com/sktime/sktime/pull/2903

Although it's only a draft PR, I would be interested to hear whether that would solve your use case.

**Comment by olerch** [comment]
Thank you for the timely response.

Unfortunately, your suggested solution #2903 doesn't seem to work. If I set the update_forecaster parameter of FittedForecaster to True the estimator parameters are updated (exactly what I don't want). If I set it to False the parameters are not updated, but the prediction is still based on the trained time series and not on the test time series.

**Reply by fkiraly** [reply]
hm, I see - so is it #4651 then?

**Reply by olerch** [reply]
Yes, exactly.

**Reply by fkiraly** [reply]
discussed today in the global forecasting workstream - have a look at the discussion in #4651, feedback appreciated. Contributions appreciated even more :-)

**Reply by jiayuanteng** [reply]
@fkiraly Hope it is okay for me to answer a question here. I have an ARIMA model trained and I have new data come in. I want to predict using the new data with the existing ARIMA model (no refitting the ARIMA model with new data). How should I implement it? Thank you very much!

**Reply by fkiraly** [reply]
I think what you may want to do is, run `update_predict` (or `update` then `predict`) with `update_params=False`.
